### DISCUSSION TITLE
Missing optional dependency "vtk" package error

### DISCUSSION THREAD

**Discussion by Tudore** [capytaine/capytaine#365] [Q&A]
Thanks again for creating this very useful tool. 

I encountered some problems recently after installing version 2. 

I started to run some cases on Capytaine v1.2 last year. The examples in the cookbook worked quite well. It was installed through conda, and I use VS code as IDE. 

Last week, I uninstalled the old version and installed the recent release, when I run the example codes, there shows a bug for this line "from capytaine.ui.vtk.animation import Animation".  The error message is "ImportError: Missing optional dependency 'vtk'. Use pip or conda to install vtk."

I checked my conda environment, the vtk package is already installed. 

To figure out the reason, I created a new environment, following: conda create --name capytaine_env --channel conda-forge Capytaine in the manual, but the error still exists. 

I noted in the discussions that this error won't affect the simulation. I wonder if you and any other users have seen such an error and if is there any solution to fix it from my end.

Thanks again.

**Comment by mancellin** [comment]
Did you try to explicitly reinstall `vtk` in your environment? It is not installed automatically with Capytaine (since as you mentioned it is not necessary for the simulation, except for 3d plotting which is not an essential part of Capytaine).
